androidApp/src/main/java/co/nimblehq/avishek/phong/kmmic/android/ui/common/AlertDialog.kt

```diff
@@ -0,0 +1,37 @@
+package co.nimblehq.avishek.phong.kmmic.android.ui.common
+
+import androidx.compose.material.*
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import co.nimblehq.avishek.phong.kmmic.android.R
+
+@Composable
+fun AlertDialog(
+    message: String,
+    onDismissRequest: () -> Unit,
+    onConfirmButtonClick: () -> Unit = onDismissRequest,
+) {
+    AlertDialog(
+        text = { Text(text = message) },
+        confirmButton = {
+            Button(
```

Comment:
Lets use `TextButton` to follow Material dialog style.
```suggestion
            TextButton(
```

Reply:
Updated in https://github.com/nimblehq/avishek-phong-kmm-ic/pull/95/commits/65f9f656b376731f85fe0294838bf6348164e320